src/sdp.ts

```diff
@@ -1,109 +1,119 @@
-import {inappropriateMultiaddr, invalidArgument, unsupportedHashAlgorithm} from './error.js';
-import {logger} from '@libp2p/logger';
-import {Multiaddr} from '@multiformats/multiaddr';
-import * as multihashes from 'multihashes';
-import {bases} from 'multiformats/basics';
+import { inappropriateMultiaddr, invalidArgument, invalidFingerprint, unsupportedHashAlgorithm } from './error'
+import { logger } from '@libp2p/logger'
+import { Multiaddr } from '@multiformats/multiaddr'
+import * as multihashes from 'multihashes'
+import { bases } from 'multiformats/basics'
 
-const log = logger('libp2p:webrtc:sdp');
+const log = logger('libp2p:webrtc:sdp')
+const CERTHASH_CODE: number = 466
 
+// Get base2 | identity decoders
 export const mbdecoder: any = (function () {
-  const decoders = Object.values(bases).map((b) => b.decoder);
-  let acc = decoders[0].or(decoders[1]);
-  decoders.slice(2).forEach((d) => (acc = acc.or(d)));
-  return acc;
-})();
+  const decoders = Object.values(bases).map((b) => b.decoder)
+  let acc = decoders[0].or(decoders[1])
+  decoders.slice(2).forEach((d) => (acc = acc.or(d)))
+  return acc
+})()
 
-const CERTHASH_CODE: number = 466;
-
-function ipv(ma: Multiaddr): string {
+// Extract the ipv from a multiaddr
+function ipv (ma: Multiaddr): string {
   for (const proto of ma.protoNames()) {
     if (proto.startsWith('ip')) {
-      return proto.toUpperCase();
+      return proto.toUpperCase()
     }
   }
-  log('Warning: multiaddr does not appear to contain IP4 or IP6.', ma);
-  return 'IP6';
-}
-function ip(ma: Multiaddr): string {
-  return ma.toOptions().host;
-}
-function port(ma: Multiaddr): number {
-  return ma.toOptions().port;
+
+  log('Warning: multiaddr does not appear to contain IP4 or IP6.', ma)
```

Comment:
is there no `log.warning` you can do instead of this poor-man's prefix?

Reply:
This is not new code from this PR.  This is a [home-rolled logger from libp2p](https://github.com/libp2p/js-libp2p-logger) and doesn't seem to have a `warn` or `warning` implementation.